Formula/abyss.rb

```diff
@@ -0,0 +1,32 @@
+class Abyss < Formula
+  desc "Genome sequence assembler for short reads"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
+  url "https://github.com/bcgsc/abyss/releases/download/2.0.2/abyss-2.0.2.tar.gz"
+  sha256 "d87b76edeac3a6fb48f24a1d63f243d8278a324c9a5eb29027b640f7089422df"
+
+  option "with-openmp", "Enable OpenMP multithreading"
+
+  needs :openmp if build.with? "openmp"
```

Comment:
almost no one is opting out (3 out of 102 installs in the last 90 days). Can we just kill the option?

Reply:
sorry I mean no one is using the option at all. I always get mpi and openmp confused.
```
Josephs-MacBook-Pro:~ joe$ brew formula-analytics --days-ago=90 abyss
install events in the last 90 days for abyss
=====================================================================================================
1 | homebrew/science/abyss                                                            |  98 |  96.08%
2 | homebrew/science/abyss --without-mpi                                              |   2 |   1.96%
3 | homebrew/science/abyss --c++11                                                    |   1 |   0.98%
4 | homebrew/science/abyss --c++11 --without-mpi                                      |   1 |   0.98%
=====================================================================================================
Total                                                                                 | 102 |    100%
=====================================================================================================
```
I don't think we need the option if no one is using it.